docs/getting-started/verify.md

```diff
@@ -32,6 +32,78 @@ The following checks were performed on each of these signatures:
 Getting the above output followed by a JSON array of payloads, ensures you the
 container image was signed using our private key.
 
+## Verify the container SBOMs
+
+We generate SBOMs for all our container image and CLI tools:
+* SBOMs for container images are attached to the corresponding image and can be found in our registry.
+* SBOMs for CLI tools are available as release assets.
+
+In this section, we will see how you can verify and inspect the SBOMs attached our container images:
+
+```bash
+# We will demo this for amd64, but it works the same for arm64.
+$ arch=amd64
+$ oras discover --platform linux/${arch} --artifact-type example/sbom ghcr.io/inspektor-gadget/inspektor-gadget:latest
+Discovered 1 artifact referencing latest
+Digest: sha256:...
+
+Artifact Type   Digest
+example/sbom    sha256:hash_of_sbom_manifest
+# As we include SBOMs in our multi architecture container image, they are also
+# signed.
+# So, let's check the SBOM is signed with our private key:
+$ cosign verify --key inspektor-gadget.pub ghcr.io/inspektor-gadget/inspektor-gadget@sha256:hash_of_sbom_manifest
+
+Verification for ghcr.io/inspektor-gadget/inspektor-gadget@sha256:hash_of_sbom_manifest
+The following checks were performed on each of these signatures:
+  - The cosign claims were validated
+  - Existence of the claims in the transparency log was verified offline
+  - The signatures were verified against the specified public key
+
+[{"critical":{"identity":{"docker-reference":"ghcr.io/inspektor-gadget/inspektor-gadget"}, ...
+]
+# Let's download the SBOM and look at it:
+$ oras pull --allow-path-traversal ghcr.io/inspektor-gadget/inspektor-gadget@sha256:hash_of_sbom_manifest
+Downloading 1d479bb51392 /tmp/gadget-container-image-linux-amd64/sbom_cyclonedx.json
+Downloaded  1d479bb51392 /tmp/gadget-container-image-linux-amd64/sbom_cyclonedx.json
+Pulled [registry] ghcr.io/eiffel-fl/inspektor-gadget@sha256:hash_of_sbom_manifest
+Digest: sha256:hash_of_sbom_manifest
```

Comment:
I tried the equivalent command like this:
```
$ oras pull --allow-path-traversal ghcr.io/eiffel-fl/inspektor-gadget@sha256:9d7fcd60f747a907f1aefa4d374ad3e61bf36bd8f4785577e477a5412ece2b6e
Downloaded empty artifact
Pulled [registry] ghcr.io/eiffel-fl/inspektor-gadget@sha256:9d7fcd60f747a907f1aefa4d374ad3e61bf36bd8f4785577e477a5412ece2b6e
Digest: sha256:9d7fcd60f747a907f1aefa4d374ad3e61bf36bd8f4785577e477a5412ece2b6e
```

But it didn't download anything in `/tmp/gadget-container-image-linux-amd64/`.